### PR TITLE
feat: Refactor cache clearing to match updated api

### PR DESCRIPTION
**Pull request by ibolton336**
- Update the UI volume handling logic to match the new API spec defined [here](https://github.com/konveyor/tackle2-hub/issues/148).

### CODE REVIEW

**Review by gildub — CHANGES_REQUESTED** [review]
Something wrong.
GET | http://localhost:9000/hub/cache/m2
returns 404 on Maven page

**Comment by ibolton336** [comment]
> Something wrong. GET | http://localhost:9000/hub/cache/m2 returns 404 on Maven page

May have to wait for the operator and hub PRs to merge before merging this @gildub

**Review by gildub — APPROVED** [review]
LGTM

**Pull request merged**
